www/js/lib/zimArchive.js

```diff
@@ -208,6 +210,10 @@ define(['zimfile', 'zimDirEntry', 'util', 'utf8'],
      */
     ZIMArchive.prototype.getTitleByName = function(titleName) {
         var that = this;
+        // If no namespace is mentioned, it's an article, and we have to add it
```

Comment:
This is not totally true, because we could have `A/A/abc` - better to always specify the namespace.